### PR TITLE
Updated maxVersions for applications

### PR DESCRIPTION
**Pull request by UtiluMark**
Part of #188
https://addons.mozilla.org/en-US/firefox/pages/appversions/

### CODE REVIEW

**Comment by UtiluMark** [comment]
@whimboo Please review and merge.

File: extension/install.rdf [hunk 1]
```diff
@@ -66,7 +66,7 @@
       <Description>
         <em:id>{92650c4d-4b8e-4d2a-b7eb-24ecf4f6b63a}</em:id>
         <em:minVersion>2.1a1pre</em:minVersion>
-        <em:maxVersion>2.25</em:maxVersion>
+        <em:maxVersion>2.41</em:maxVersion>
```
Comment (xabolcs) [inline extension/install.rdf]: FYI.

In general Firefox version x == SeaMonkey version 2.x+3, but for now SM version was left behind.

**Comment by xabolcs** [comment]
r+

**Comment by whimboo** [comment]
I merged this PR as https://github.com/mozilla/nightlytt/commit/a8e175e01291083882e745e418e2dca84fc3ba18. Thank you both for letting me land it right away.

**Pull request closed without merging**